assets/sass/components/_forms.scss

```diff
@@ -904,3 +904,23 @@ $inputOutline: 3px solid get-color('border', 'outline');
         }
     }
 }
+
+.js-form-warn-unsaved.js-session-form {
```

Comment:
Ooh. we need to re think this CSS. Best to avoid handing any styling off a `js-` prefixed class, and this is far too specific. I'd add a modifier to form actions and make the `last-save-timestamp` a child element. The text based on the mockup should be link blue, and only the timestamp emboldened. I'd suggest something like this:

```scss
.form-actions--flex {
    display: flex;
    flex-flow: row wrap;
    justify-content: space-between;
    align-items: center;

    .form-actions__primary,
    .form-actions__secondary {
        flex: 0 0 auto;
    }

    .form-actions__timestamp {
        margin-bottom: 0;
        color: get-color('links', 'base');
        font-size: 15px;

        time {
            font-weight: font-weight('body', 'semibold');
        }
    }
}
```

With the following markup:

```
<div class="form-actions form-actions--flex">
    <div class="form-actions__primary">
        <input class="btn" type="submit" value="{{ copy.continue }}"/>
    </div>
    <p class="form-actions__secondary form-actions__timestamp">
        Last successful save - <time datetime="{{ lastSaveDatetime }}">{{ lastSaveTime }}</time>
    </p>
</div>
```

Reply:
`lastSaveDatetime` would be the ISO datetime value